feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -254,6 +257,33 @@ func (tc *testCase) transferOS(ctx context.Context, t *testing.T, standby bool)
 	}
 }
 
+func (tc *testCase) isSupervisorReady(ctx context.Context, t *testing.T) error {
+	var counter int8
+	const maxRetries = 15
+	for i := int8(1); i < maxRetries; i++ {
+		if counter == maxRetries {
+			return fmt.Errorf("OS.Verify RPC did not report the standby supervisor state as available")
```

Comment:
t.Fatal instead of fmt.Errorf?

Reply:
I suppose I could make this function not return anything and simply return t.Fatal, is that the standard for these functions in this testcase? If so, I'll make the change